conda_smithy/ci_register.py

```diff
@@ -364,6 +364,12 @@ def add_conda_forge_webservice_hooks(user, repo):
             ]
         ),
         get_conda_hook_info(
+            "http://conda-forge.herokuapp.com/conda-forge-feedstocks/hook",
+            [
+                "push"
```

Comment:
Do we need the same event sent to the same application twice? We can do both things in `conda-forge-webservices` using one hook right?

Reply:
It's a good point. Would be good to silo them from each other a bit though.

Reply:
I'm fine with either way.